src/Client.php

```diff
@@ -1210,4 +1246,55 @@ public function setCookie($cookieName, $cookieValue=null)
         $soapClient->__setCookie($cookieName, $cookieValue);
         return $this;
     }
+
+    /**
+     * @return boolean
+     */
+    public function getKeepAlive()
+    {
+        return $this->keepAlive;
+    }
+
+    /**
+     * @param boolean $keepAlive
+     * @return self
+     */
+    public function setKeepAlive($keepAlive)
+    {
+        $this->keepAlive = (bool) $keepAlive;
+        return $this;
+    }
+
+    /**
+     * @return int
+     */
+    public function getSslMethod()
+    {
+        return $this->sslMethod;
+    }
+
+    /**
+     * @param int $sslMethod
+     * @return self
+     */
+    public function setSslMethod($sslMethod)
+    {
+        if (!in_array($sslMethod, static::$supportedSslMethods, true)) {
+            throw new Exception\InvalidArgumentException(sprintf(
+                'Invalid SSL method specified. Use one of %s constants.',
+                'SOAP_SSL_METHOD_TLS, SOAP_SSL_METHOD_SSLv2, SOAP_SSL_METHOD_SSLv3, SOAP_SSL_METHOD_SSLv23'
+            ));
+        }
+
+        $this->sslMethod = $sslMethod;
+        return $this;
+    }
+
+    /**
+     * @return int[]
+     */
+    public static function getSupportedSslMethods()
```

Comment:
For to check if constant list become outdated. See test case.

Probably this could be removed and property ready using reflection